### ISSUE TITLE
Add Current Situation to documentation (1.2.1)

### ISSUE THREAD

**Issue by PinkSylvie** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#56]
Add the information needed to section mentioned above following the explanation provided by the professor below. 
• Try to use this section to prepare for the next section of
    needs. For example in a project concerned with used car owner/repair
    history you could emphasize the point that there is a large used car
    market and that for used cars information about the car's history is
    more important than for new cars.
  • If you have to make reference to data like percentages or estimates,
    its good if you include where you found it or a source.
    
Scoring:
Difficulty: 1
Urgency: 4

**Comment by AngelCIICMorales** [comment]
Add score.

**Comment by rukitoyordan** [comment]
Dominoes has long been a very traditional game for family gatherings and casual hangouts, especially in Puerto Rico and Latin American countries. People of all ages gather around to share an experience and talk about their lives. It is not at all about winning but rather of the connecting memories that are cherished throughout everyone. However, when it comes to playing dominoes online, the options are somewhat limited and do not capture the Puerto Rican essence, if you will.. A lot of the charm of dominoes is lost when you can’t bond or strategize with friends in real-time. As more classic games transfer to the online realm, there’s a clear gap when it comes to providing an authentic multiplayer dominoes experience that feels as lively and fun as playing in person. This makes a situation definite on why this could be done.

It’s important to keep the cultural connection that is shared by all Puerto Ricans alive. Making some variables of that Puerto Rican uniqueness, such as historical and symbolic designs or art styles, could definitely better players to feel more connected to their roots. This generates connection, and gives way into solving other situations that may not be met in the gaming market.

In summary, there’s a clear-cut situation for the market on a multiplayer dominoes platform that basically captures the essence of the game. Players want more than just a basic trip down memory lane, but rather they’re looking for an experience that feels social, culturally rich, and engaging. By building a platform that allows for real-time multiplayer games, we can create a space where both casual and competitive players will feel at home. This will not only attract those who already love dominoes but also bring in new players who are looking for a fresh, engaging way to experience the game.

**Comment by rukitoyordan** [comment]
This has been added already to the M1 Documentation accordingly.

**Comment by AngelCIICMorales** [comment]
LGTM.
